src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_WellKnownAttributes.cs

```diff
@@ -6318,9 +6421,12 @@ public class Base<T> {}
 public class Derived : Base<Base<int>> {}
 ";
             CreateCompilation(source).VerifyDiagnostics(
+                // (27,19): warning CS0612: 'SomeType' is obsolete
```

Comment:
>warning CS0612: 'SomeType' is obsolete [](start = 28, length = 38)

I didn't understand why we previously didn't report any diagnostics on usage of `SomeType`, but now we do for this one usage.

Reply:
> I didn't understand why we previously didn't report any diagnostics on usage of SomeType, but now we do for this one usage.

Do you mean you didn't or you don't?

It's because only the accessor is obsolete, but we use an obsolete return type for the property, which is not obsolete. An obsolete accessor does not suppress warnings about using an obsolete return type.

Reply:
Thanks you answered my question (details of suppression behavior).